callhandle.py

```diff
@@ -0,0 +1,52 @@
+def handle_call(members, all_flights): # Two string arrays. Members contain all members of one flight, eg {"VIKING 11", "VIKING 12"}. all_flights contains all flights currently active, eg. ({"VIKING 11", "VIKING 12"}, {"DARKSTAR", "PYTHON 11", "PYTHON 21", "VIKING 21", "VIKING 22"})
```

Comment:
comments are _almost always_ a bad thing to add, if you feel a comment is warranted that is instead a sign something else should be improved instead. (There are some exceptions, but they are exceedingly rare, such as linking some math algorithm, or explaining why you are doing something super weird on purpose)

For example, in this case you mention `# Two string arrays.`, instead, add type hints.

```python
def handle_call(members: list[str], all_flights: list[str]) -> str:
    ...
```

Reply:
`handle_call` is also a very _non_-descriptive name. Always try to choose descriptive names.

Something like `handle` might be okay if it was wrapped in a class e.g. `FooBuilder.build()` but in this case a callhandle.handle_call is saying basically nothing.